client/apps/stripe-connect-account/view-wrapper.js

```diff
@@ -47,13 +49,23 @@ class StripeConnectAccountWrapper extends Component {
 	render() {
 		const {
 			isLoading,
+			isOAuthInitializing,
 			stripeConnectAccount,
+			stripeError,
 			isDeauthorizing,
 			isReloading,
 			oauthURL,
 			translate,
 		} = this.props;
 
+		if ( ! isOAuthInitializing && stripeError ) {
```

Comment:
If this were moved below the other conditionals, would checking for `stripeError` be enough?